### PR TITLE
DashScope: Switch to another available image url for WanxImageModelIT

### PR DESCRIPTION
**Pull request by jiangsier-xyz**
## Change
The previous image link could not be processed properly. Replace to another available one.
And, more friendly error messages are provided when the image generation fails.

## General checklist

- [X] There are no breaking changes
- [ ] I have added unit and integration tests for my change
- [X] I have manually run all the unit and integration tests in the module I have added/changed, and they are all green
- [ ] I have manually run all the unit and integration tests in the [core](https://github.com/langchain4j/langchain4j/tree/main/langchain4j-core) and [main](https://github.com/langchain4j/langchain4j/tree/main/langchain4j) modules, and they are all green

### CODE REVIEW

File: langchain4j-dashscope/src/main/java/dev/langchain4j/model/dashscope/WanxImageModel.java [hunk 1]
```diff
@@ -99,7 +100,14 @@ public Response<Image> edit(Image image, String prompt) {
 
         try {
             ImageSynthesisResult result = imageSynthesis.call(builder.build());
-            return Response.from(imagesFrom(result).get(0));
+            List<Image> images = imagesFrom(result);
+            if (images.isEmpty()) {
+                ImageSynthesisOutput output = result.getOutput();
+                String errorMessage = String.format("[%s] %s: %s",
+                        output.getTaskStatus(), output.getCode(), output.getMessage());
+                throw new NullPointerException(errorMessage);
```
Comment (Wadimz) [inline langchain4j-dashscope/src/main/java/dev/langchain4j/model/dashscope/WanxImageModel.java]: hey @jiangsier-xyz! curious if this case is really NPE. Maybe we can throw better exception here? thanks!

Comment (jiangsier-xyz) [reply langchain4j-dashscope/src/main/java/dev/langchain4j/model/dashscope/WanxImageModel.java]: > hey @jiangsier-xyz! curious if this case is really NPE. Maybe we can throw better exception here? thanks!

I've thought about this，but have no good idea. What are your suggestions?

Comment (Wadimz) [reply langchain4j-dashscope/src/main/java/dev/langchain4j/model/dashscope/WanxImageModel.java]: sorry for delayed response. I noticed project tend to use IllegalStateException for result errors

Comment (jiangsier-xyz) [reply langchain4j-dashscope/src/main/java/dev/langchain4j/model/dashscope/WanxImageModel.java]: > sorry for delayed response. I noticed project tend to use IllegalStateException for result errors

OK, that makes sense.

**Review by Wadimz — APPROVED** [review]

**Review by Wadimz — APPROVED** [review]

**Review by Wadimz — APPROVED** [review]

**Review by Wadimz — APPROVED** [review]

**Review by ghost — APPROVED** [review]
@jiangsier-xyz thank you!

**Pull request merged**